### DISCUSSION TITLE
Low LCP score on mobile

### DISCUSSION THREAD

**Discussion by NicolasCassunde** [vercel/next.js#68558] [Help]
### Summary

I'm experiencing a low Largest Contentful Paint (LCP) score on mobile devices. The PageSpeed score is 100% on desktop but drops to 91% on mobile.

Here's the relevant code snippet:

`     <Image
          src={data.feature_image as string}
          alt={data.title as string}
          priority
          width={384}
          height={384}
          className="object-cover"
        />
`

![image](https://github.com/user-attachments/assets/9704cc47-cb3c-490d-9bc3-d8b8cabdc8d8)

**Comment by brodsolutions** [comment]
Same here, only on Mobile. 
`<Image
                  alt=""
                  src={process.env.NEXT_PUBLIC_BASE_URL + "/uploads/BROD_SOLUTION_BRANDING_KIT_20_LOCKUPS_Floppy_Disk.png"}
                  width={275}
                  height={275}
                  quality={100}
                  priority
                  sizes="275px"
                />`

I thought NextImage would take care of prefetching to resolve this issue. I may try to add a <head> prefetch request manually to see if that works.

**Comment by bartekczyz** [comment]
I'm dealing with the same problem, also only on mobile.

**Comment by HakanSungur** [comment]
Same problem for me. Did you find any solution?

**Comment by Gavrushuk** [comment]
Same problem for me on mobile. The desktop has perfect scores
